src/observables/ForkJoinObservable.ts

```diff
@@ -9,7 +9,7 @@ export class ForkJoinObservable<T> extends Observable<T> {
                                     Promise<any> |
                                     ((...values: Array<any>) => any)>) {
      super();
-   }
+  }
```

Comment:
When @staltz reads code : 

![](https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcTgIax2lVjcNaxIlWdouM9D4UXWchAVEJM_yID3l5lHNPgwvQhu_A)

I'll get it updated :)